jfrog-oauth/README.md

```diff
@@ -17,15 +17,16 @@ Install the JF CLI and authenticate package managers with Artifactory using OAut
 ```tf
 module "jfrog" {
   source         = "registry.coder.com/modules/jfrog-oauth/coder"
-  version        = "1.0.15"
+  version        = "2.0.0"
```

Comment:
yes, we will go with 1.0.19 for now until we resolve #157.

Reply:
Ok. I was bumping the major version since the input variable format for the repos is a breaking change, but if there's an outside constraint, I'll update the readme to reflect the target version of 1.0.19

Reply:
It is indeed a breaking change but we do not want to bump all future modules to 2.0.0 before we do #157.